test/nmodl/transpiler/integration/mod/glia_matexp.mod

```diff
@@ -0,0 +1,37 @@
+: Example of MOD file using matexp solver
+
+NEURON {
+    SUFFIX glia
+    RANGE Aalfa, Valfa, Abeta, Vbeta
+}
+
+PARAMETER {
+    Aalfa = 353.91 ( /ms)
+    Valfa = 13.99 ( /mV)
+    Abeta = 1.272  ( /ms)
+    Vbeta = 13.99 ( /mV)
+    n1 = 5.422
+    n4 = 0.738
+}
+
+STATE {
+    C1
+    C2
+}
+
+BREAKPOINT {
+    SOLVE kstates METHOD matexp
+}
+
+FUNCTION alfa(v(mV)) {
+    alfa = Aalfa*exp(v/Valfa)
+}
+
+FUNCTION beta(v(mV)) {
+    beta = Abeta*exp(-v/Vbeta)
+}
+
+KINETIC kstates {
+    ~ C1 <-> C2 (n1*alfa(v),n4*beta(v))
```

Comment:
I don't see an INITIAL block so C1 and C2 are initialized to 0 and never change during simulation. On the other hand, if you add a CONSERVE statement, there will be only one state variable. 

I guess it might be easiest just  to introduce 
```
INITIAL {
    C1 = 1
    C2 = 0
}
```

Reply:
The purpose of this file is to test that the translated nmodl files can be compiled. This file should not be visible to the end user. 

I will add an initial block and conserve statement so that this tests those additional code-paths.